viewing_party/party.py

```diff
@@ -1,23 +1,206 @@
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+
+   
+    movie = {
+        "title":title,
+        "genre":genre,
+        "rating":rating
+    }
+    #returns none if title or movie or rating is falsy
+    if title == None or genre == None or rating == None:
+        return None 
+    
+    else: 
+        return movie
+
+
+
+
+
+
+def add_to_watched(user_data, movie):
+    #user data is dictionary w/ key "watched"
+    
+    user_data = {
+        "watched": [movie]
+    }
+
+    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    #user_data {"watchlist": list of dictionaries representing the movies the user wants to watch}
+ 
+    user_data = {"watchlist":[movie]}
+
+    return user_data

```

Comment:
What if the "watched" and "watchlist" are already populated with some movies? `.append` is the best choice of operation here.